src/dotnet/commands/dotnet-help/HelpCommandParser.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.DotNet.Cli.CommandLine;
+
+namespace Microsoft.DotNet.Tools.Help
+{
+    internal static class HelpCommandParser
+    {
+        public static Command Help()
+        {
+            return Create.Command(
+                "help",
+                LocalizableStrings.AppFullName,
+                Accept.ZeroOrOneArgument()
+                    .With(
+                        LocalizableStrings.CommandArgumentDescription,
+                        LocalizableStrings.CommandArgumentName),
+                Create.Option(
+                    "-h|--help|-?|/?",
```

Comment:
It is already there
Here is a test for it
https://github.com/dotnet/cli/blob/d347e2781d89c30385c09df143fd48fb60fd3932/test/dotnet-help.Tests/GivenThatIWantToShowHelpForDotnetCommand.cs#L61

Share I remove it along with the test?